data/json/recipes/ammo/grenades.json

```diff
@@ -14,7 +14,7 @@
     "charges": 1,
     "using": [ [ "ammo_40mm", 1 ] ],
     "tools": [ [ [ "press", -1 ] ] ],
-    "components": [ [ [ "gunpowder", 30 ] ], [ [ "combatnail", 25 ] ] ]
+    "components": [ [ [ "gunpowder", 30 ], [ "gunpowder_pistol", 30 ], [ "chem_black_powder", 30 ] ], [ [ "combatnail", 25 ] ] ]
```

Comment:
Wouldn't blackpowder loads need to be a completely separate thing? They would create fouling that needs cleaning and smoke clouds, for instance.

Reply:
Oh, that's true.